cpg_utils/hail_batch.py

```diff
@@ -164,6 +165,35 @@ def full_path(self, prefix: str, suffix: str) -> str:
         return os.path.join(f'{self.scheme}://', prefix, suffix)
 
 
+class Namespace(Enum):
+    """
+    Storage namespace.
+    https://github.com/populationgenomics/team-docs/tree/main/storage_policies#main-vs-test
+    """
+
+    MAIN = 'main'
+    TEST = 'test'
+
+    @staticmethod
+    def parse(str_val: str) -> 'Namespace':
+        """
+        Parse value from a string.
```

Comment:
This isn't so much parsing an enum string representation, but mapping an access level to a namespace, right? Might be worth clarifying that in the method name / documentation.